flow/designs/tsmc65lp/coyote/config_hier_tmp.mk

```diff
@@ -0,0 +1,6 @@
+include $(dir $(DESIGN_CONFIG))/config_hier.mk
+
+export FLOW_VARIANT = hier_tmp
+unexport RTLMP_FLOW
+
+export PLACE_DENSITY         = 0.60
```

Comment:
Why run this at a different density if you want to have a comparison with rtlmp?